QTFK.Extensions/Objects/Factory/GenericFactory.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace QTFK.Extensions.Objects.Factory
+{
+    public static class GenericFactory
+    {
+        /// <summary>
+        /// Creates a new instance of <typeparamref name="T"/> that receives all public instance properties from <paramref name="source"/>
+        /// </summary>
+        /// <typeparam name="T"><paramref name="source"/> and returned type</typeparam>
+        /// <param name="source">Instance that contains the original properties that will be carried over the new intance</param>
+        /// <returns>The created instance</returns>
+        /// <remarks>Intended to be used over POCO's and objects that represent simple models.
+        /// Properties that implement special logic inside their getter/setter's may present undesired results.
+        /// Members are NOT cloned, only properties, which may indirectly affect members.</remarks>
+        public static T ShallowClone<T>(this T source) where T : class, new()
```

Comment:
This method should call the other `ShallowClone<T>` method with default parameters like:
```cs
return ShallowClone<T>(source, p => p.CanRead && p.CanWrite, BindingFlags.Instance | BindingFlags.Public);
```

Making this, you do not need `GetPublicInstanceProperties` private method ^__^